pkg/objects/context.go

```diff
@@ -0,0 +1,70 @@
+package objects
+
+import (
+	"context"
+)
+
+// FileContextKey is the context key for the files
+var FileContextKey = &ContextKey{"files"}
```

Comment:
may want to put a comment in here (or the readme) indicating the difference between the input key provided by the user and the context key we use